textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/keywordmatch/KeywordMatcherTest.java

```diff
@@ -175,15 +126,15 @@ public void testKeywordMatcher() throws Exception {
     @Test
     public void testSingleWordQueryInStringField() throws Exception {
         //Prepare Query
-        String query = "Bruce";
+        String query = "bruce";
```

Comment:
Understood. It will be good to add this explanation as a comment to the codebase.